src/test/unit/config_eeprom_unittest.cc

```diff
@@ -99,17 +100,17 @@ const pgRegistry_t __pg_registry[] =
         .pgn = 1,
         .format = 0,
         .flags = PGC_PROFILE
-    },
-    {
-        .base = nullptr,
-        .ptr = 0,
-        .size = 0,
-        .pgn = 0,
-        .format = 0,
-        .flags = 0
-    },
+    }
 };
 
+const pgRegistry_t __pg_registry_end[] = {}; // FIXME enforce the location of this somehow.
```

Comment:
@ledvinap this is specifically what i do not like about this solution currently.